adobecom/milo
# MWPW-140468: M@S Lana - clientId and error rate

Lana fixes for commerce
* add 'merch-at-scale' client id to commerce related lana errors
* set error rate to 30 (was 100)

Resolves: [MWPW-140468](https://jira.corp.adobe.com/browse/MWPW-140468)

To verify the change:

- add env=prod parameter
- block any WCS fetch request
- reload page and in network search for lana
- observe lana request with 'merch-at-scale' clientId and changed rate parameter

before
<img width="966" alt="image" src="https://github.com/adobecom/milo/assets/6617234/7d601308-9aa1-4c30-972c-3f83e560b9cc">
after
<img width="646" alt="image" src="https://github.com/adobecom/milo/assets/6617234/ccc4e281-9d63-4d7a-82fb-efe747cf3412">


**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/mariia/testpagenomep?martech=off
- After: https://mas-lana--milo--3ch023.hlx.page/drafts/mariia/testpagenomep?martech=off


[taco pr](https://git.corp.adobe.com/wcms/tacocat.js/pull/490)
